byond-extools/src/monstermos/monstermos.cpp

```diff
@@ -17,7 +18,7 @@ trvh fuck(unsigned int args_len, Value* args, Value src)
 
 std::unordered_map<std::string, Value> gas_types;
 std::unordered_map<unsigned int, int> gas_ids;
-//std::unordered_map<unsigned int, std::shared_ptr<GasMixture>> gas_mixtures;
+std::unordered_set<unsigned int> modified_datums;

```

Comment:
Would a hashmap not be faster, than iterating over a set?

Reply:
unordered_set is literally a hashmap, `modified_datums.find` will thus be O(1)